pkg/github/issues.go

```diff
@@ -49,6 +53,8 @@ func (c Issues) Frames() data.Frames {
 			v.Title,
 			v.Author.User.Login,
 			v.Author.User.Company,
+			fmt.Sprintf("%s/%s", v.Repository.Owner.Login, v.Repository.Name),
```

Comment:
```suggestion
			v.Repository.NameWithOwner,
```

Just remembered that Repository has a field that does this for us

https://github.com/grafana/github-datasource/blob/9a25a6d52d11f2bd963da49dc25117ecfa2bdd3f/pkg/github/repositories.go#L39